### DISCUSSION TITLE
Resetting circle diameter once animation ends

### DISCUSSION THREAD

**Discussion by ejwm** [bonsai-rx/bonsai#1962] [Q&A] [answered]
[2vid_loom_repeat.zip](https://github.com/user-attachments/files/16532791/2vid_loom_repeat.zip)
Hi,
I'm fairly new to Bonsai and am looking for a little help.
I am trying to create a looming stimulus. The stimulus starts with 10s with a blank screen, then holds on the circle for 10s, then I have animated a circle from BonVision to change from diameter 0.1 to 2.5 over a period of time. However, this means that when I run the file again the circle starts at diameter 2.5 rather than resetting to 0.1. Is there any way to reset?
Thanks,
Eira
![Screenshot 2024-08-07 194421](https://github.com/user-attachments/assets/66b6d034-3780-490b-90bf-8b058d8e0f91)

**Comment by bruno-f-cruz** [comment, marked as answer]
Remove the SubscribeWhen connected to RangeAnimation. Alternatively, you can also just send a single Float value at the start of the sequence by using the Merge operation after RangeAnimation.

Hope it helps!

**Reply by ejwm** [reply]
Thanks for answering! Sending the float and merging did exactly what I needed
